ecs-task.cfndsl.rb

```diff
@@ -172,6 +174,7 @@
         Policies(policies)
       end
 
+      secrets = external_parameters.fetch(:secrets, [])
```

Comment:
secrets doesn't come from config, it's defined up the top and added to during the task def secrets